docs/README.md

```diff
@@ -357,7 +361,7 @@ If the vendor label in the `OWNERS` file is wrong, you can follow the [partner
 connect documentation][partners] to update the `OWNERS` file.
 
 If the directory structure (organization name) is wrong, please reach out to
-[Technology Partner Success Desk][partner-success-desk].
+[Technology Partner Success Desk][partner-acceleration-desk].
```

Comment:
I know this PR is already merged, but should this also be changed to "Partner Acceleration Desk" ?

Reply:
I opened https://github.com/openshift-helm-charts/development/pull/285 to fix this.